Add warning about schema designer in docs (#500)
graknlabs/docs: 07-workbase/04-schema-designer.md

toc: false
---

<div class = "note">
[Warning]

> The schema designer is currently not yet stable in 2.0, and it has been disabled since the release 2.0.0-alpha-3. To view your schema in the visualiser, please run the following query:

Reviewer: ```suggestion
The schema designer is not yet stable in version 2.0 and has been disabled as of 2.0.0-alpha-3. To view your schema in the visualiser, please run the following query:
```
